hapi-fees/src/main/java/com/hedera/services/usage/token/TokenCreateUsage.java

```diff
@@ -65,7 +65,7 @@ public FeeData get() {
 		usageEstimator.addBpt(baseSize);
 		usageEstimator.addRbs(baseSize * lifetime);
 		addNetworkRecordRb(BASIC_ENTITY_ID_SIZE);
-		addTokenTransfersRecordRb(1, 1);
+		addTokenTransfersRecordRb(1, 1, 1);
```

Comment:
Depending on whether there is `initialSupply` or not, we are adding token transfers record. If there is `initialSupply` set, it must be `1,1,0` if there is no `initialSupply` set, it should be `1,0,0` 

When we are creating NON FUNGIBLE token types, we are not adding transaction records for nft transfer

Reply:
Corrected!